File: android/app/src/main/java/com/tisza/esemenynaptar/Event.java
package com.tisza.esemenynaptar;

public class Event
{
	private final Category category;
	private final String text;

	public Event(Category category, String text)
	{
		this.category = category;
		this.text = text;
	}

	public Category getCategory()
	{
		return category;
	}

	public String getText()
	{
		return text;
	}
}

File: android/app/src/main/java/com/tisza/esemenynaptar/database/Event.kt
package com.tisza.esemenynaptar.database

import androidx.room.*
import com.tisza.esemenynaptar.*
import java.util.*

@Entity(tableName = "event", primaryKeys = ["year", "month", "day", "category"])
class Event(var year: Int, var month: Int, var day: Int, var category: Category, var text: String, var isLiked: Boolean) 

val Event.calendar : Calendar
    get() {
        val cal = Calendar.getInstance()
        cal.set(year, month, day)
        return cal
    }
